packages/mds-types/index.ts

```diff
@@ -262,6 +334,23 @@ export const TAXI_STATE_EVENT_MAP = TaxiStatusEventMap({
   stopped: ['trip_stop', 'reservation_stop']
 })
 
+export const TNC_STATE_EVENT_MAP = TncStatusEventMap({
+  available: [
+    'driver_cancellation',
+    'enter_jurisdiction',
+    'passenger_cancellation',
+    'provider_cancellation',
+    'service_start',
+    'trip_end'
+  ],
+  reserved: ['reservation_start', 'comms_restored', 'enter_jurisdiction'],
```

Comment:
Shouldn't `passenger_cancellation` and `provider_cancellation` be in there too?

Reply:
I think this is the opposite way from what you're thinking, this is basically the events that can get you _into_ this state, not out of it. `passenger_cancellation` and `provider_cancellation` get you into `available` (which is up above)

Reply:
Ah I see what you mean, I think the `TNC_EVENT_STATES_MAP` from way above is a little off, I'll patch that up

Reply:
I ended up just whacking this data-type, it's very unintuitive to maintain with the new 'multi-trip state machine within the vehicle state machine' pattern, and wasn't used _anywhere_.